doc/sphinx/eduro.rst

```diff
@@ -0,0 +1,35 @@
+Robot Eduro
+===========
+
+Eduro was designed and created by Jan Roubicek back in 2007 as EDUcational
+RObot for the university. The electronics was manufactured by Tomas Roubicek,
+and it includes CAN network modules using CANopen protocol, custom made stepper
+motor drivers with encoders (SMAC) and power management.
+
+The original "small" Eduro was rebuild into outdoor version "Eduro Maxi" in
+2010. Since then it participated in many outdoor competitions and gained
+several prices including 1st place in Field Robot Event and Robotour contests.
+
+The robot is differentially driven with two motors. It is tricycle with very
+high maneuverability. The maximal speed is limited to 1 meter per second.
+
+The communication uses SYNC signal, after which all CANopen modules transmit
+their status. The default is 20Hz cycle, but can be modified. The encoder
```

Comment:
Who keeps the time? Does some module send SYNC packets or the bridge or the PC?

Reply:
Please note it in the rst, thanks. I think it would be great if we knew how this works.

Reply:
OK added footnote regarding SYNC generation